cpp/velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -364,9 +364,11 @@ class SubstraitToVeloxPlanConverter {
       }
     }
 
-    // Set a value for the not(equal) condition.
-    void setNotValue(const std::optional<variant>& notValue) {
-      notValue_ = notValue;
+    // Set a list of values to be used in the push down of 'not in' expression.
+    void setNotValue(const std::vector<variant>& notValue) {
+      for (const auto& value : notValue) {
+        notValues_.emplace_back(value);
+      }
```

Comment:
nit: suggestion on the namings: notValue -> notValues, setNotValue -> setNotValues

Reply:
done